server/tests/models/users.spec.js

```diff
@@ -1,4 +1,4 @@
-import chai, { assert } from "chai";
+import chai, { assert, expect } from "chai";
```

Comment:
But expect is not being used here, why are you importing it 🤷‍♂️

Reply:
expect is used in the test.

Reply:
seen it never mind